tests/drivers/uart/uart_elementary/Kconfig

```diff
@@ -8,4 +8,11 @@ config DUAL_UART_TEST
 config SETUP_MISMATCH_TEST
 	bool "Enable mismatched configuration in dual UART test"
 
+config UART_BAUDRATE_MISMATCH
+	int "UART baudrate for DUT_AUX to test mismatch case"
+	default 9600
+	help
+	  UART baudrate for DUT_AUX that is intentionally different from DUT.
+	  Used when SETUP_MISMATCH_TEST is enabled.
```

Comment:
Do you mind making UART_BAUDRATE_MISMATCH explicitly dependent on SETUP_MISMATCH_TEST?

Reply:
good idea